teemaderegister-react/src/components/SupervisorMeta.jsx

```diff
@@ -76,13 +77,15 @@ class SupervisorMeta extends PureComponent {
       data: {
         profile: {
           firstName,
-          lastName
+          lastName,
+          description
         }
       }
     } = this.props
 
     const name = `${firstName} ${lastName}`
     const imgSrc = `http://via.placeholder.com/150/b1e3da/ffffff/?text=${firstName[0]} ${lastName[0]}`
+    const descriptionBox = `${description}`
```

Comment:
no need for this line

Reply:
When this line is removed, 
![image](https://user-images.githubusercontent.com/55129245/104852409-9389fd00-5903-11eb-96c2-79756dc0954e.png)

That page stops working and the following error is displayed:
![image](https://user-images.githubusercontent.com/55129245/104852402-8b31c200-5903-11eb-8433-0edae61f8d16.png)

Reply:
it should be then {description} used later, also wrote to slack